src/Symfony/Component/Routing/Loader/AnnotationDirectoryLoader.php

```diff
@@ -76,6 +76,15 @@ function (\SplFileInfo $current) {
      */
     public function supports($resource, $type = null)
     {
+        if ($type && 'annotation' !== $type) {
+            return false;
+        }
+
+        // classes are handled by the AnnotationClassLoader
+        if (false !== strpos($resource, '\\') || class_exists($resource)) {
```

Comment:
that's not good. Importing `%kernel.project_dir%/src/Controller` will have backslashes in it on Windows AFAICT.

Reply:
so we should add the regexp check also